daml-foundations/daml-ghc/tests/GenericTemplates.daml

```diff
@@ -10,17 +10,18 @@ module GenericTemplates where
 import Prelude hiding (Template (..), Choice (..), create, fetch, exercise)
 
 class Template t where
+  -- TODO(MH): Decide if we actually want to provide `signatory` and
+  -- `observer` to DAML devs. If not, we can remove them here since they are
+  -- not required for template desugaring/resugaring anymore.
   signatory : t -> [Party]
   observer : t -> [Party]
-  observer _ = []
-
-class Template t => Creatable t where
   create : t -> Update (ContractId t)
   fetch : ContractId t -> Update t
 
 class Template t => Choice t c r | t c -> r where
+  -- TODO(MH): Decide if we actually want to provide `choiceController` and
+  -- `action` to DAML devs. If not, we can remove them here since they are
+  -- not required for template desugaring/resugaring anymore.
```

Comment:
I don't think we do - we have never advertised them and they are slightly odd names.